hudi-common/src/test/java/org/apache/hudi/common/testutils/HoodieTestUtils.java

```diff
@@ -237,11 +229,12 @@ public static void createPendingCleanFiles(HoodieTableMetaClient metaClient, Str
 
   public static void createCorruptedPendingCleanFiles(HoodieTableMetaClient metaClient, String commitTime) {
     Arrays.asList(HoodieTimeline.makeRequestedCleanerFileName(commitTime),
-        HoodieTimeline.makeInflightCleanerFileName(commitTime)).forEach(f -> {
+        HoodieTimeline.makeInflightCleanerFileName(commitTime))
+        .forEach(f -> {
           FSDataOutputStream os = null;
           try {
             Path commitFile = new Path(
-                    metaClient.getBasePath() + "/" + HoodieTableMetaClient.METAFOLDER_NAME + "/" + f);
+                metaClient.getBasePath() + "/" + HoodieTableMetaClient.METAFOLDER_NAME + "/" + f);
```

Comment:
Can we use `Paths.get()` to reduce link path parts via `/`?

Reply:
sure, fixed 2 of this. Please note that, as this is from the original codebase, i didn't change all of this kind of usage in this file. I think it could be a good chance to change while moving the APIs to `HoodieTestTable`.

Reply:
Ok, I noticed this because I saw the indent has been changed. Actually, I suggested that we can focus on the core work about the PR want to do. Do not touch something else e.g. indent and code style. Of cause, I should not ask you to change this.

Reply:
ok @yanghua ,  understood. I agree that the work should focus on the task. I didn't intend to change these; intellij did it for me. i should have disabled the auto-formatting. will keep this sort of diffs out from now on. thanks